source/onboard/sso-gitlab.rst

```diff
@@ -50,7 +50,10 @@ Step 1: Add an OpenID Connect application to your GitLab account
 
    If your GitLab instance is not set up to use SSL, your URIs must begin with ``http://`` instead of ``https://``.
 
-  c. Select the scopes: ``openid``, ``profile``, and ``email``.
+  c. Select scopes.
+  
+     - For Mattermost Team Edition, select "read_user".
```

Comment:
@sbishel - Do we want to specify the Mattermost releases this applies to? Based on the original issue, this may have been a change between ~v5.30 and v5.34?

Reply:
Pretty sure this applies to all editions as Team Edition only supports Gitlab OAuth.